frontend/src/models/devices.ts

```diff
@@ -1,6 +1,7 @@
 import { graphQLFetch, graphQLAdaptor } from '../services/graphQL'
 import { createModel } from '@rematch/core'
 import { r3 } from '../services/remote.it'
+import { store } from '../store'
```

Comment:
You can get the global state from the second parameter in the effect below... I'll comment below.